test/e2e/preset_test.go

```diff
@@ -846,6 +845,26 @@ var _ = Describe("Workspace Preset", func() {
 		validateWorkspaceReadiness(workspaceObj)
 	})
 
+	It("should create a llama-3.1-8b-instruct workspace with preset public mode successfully", func() {
```

Comment:
The default e2e test in pr will only run these fastcheck cases. You can add a temporary label here to force run it once.